src/core/engines/Cesium/Feature/Box/Edge.tsx

```diff
@@ -0,0 +1,73 @@
+import { ArcType, Cartesian3, Color, TranslationRotationScale } from "cesium";
+import { FC, memo } from "react";
+import { Entity, PolylineGraphics } from "resium";
+
+import { EventCallback } from "@reearth/util/event";
```

Comment:
Define types inside core module.

- Only used in cesium -> `engines/Cesium/types.ts`
- Mainly used in EngineRef -> `Map/types/*.ts`
- Mainly used in mantle -> `mantle/types/*.ts`

Reply:
I fixed these import. [fix: import modules under reearth/core](https://github.com/reearth/reearth-web/pull/377/commits/c3028411f73bc2f5ef7827a93d5120d4486d3cf6)